### PR TITLE
Add logo and update pip install requirements

### PR DESCRIPTION
**Pull request by ajlee21**
This PR does the following:

1. Adds possible logo to the ReadMe -- any thoughts?
2. Addresses the comment from the previous PR that I overlooked: https://github.com/greenelab/ponyo/pull/10#discussion_r450875380

Since this is a pip installable package, I have updated the dependencies listed in the setup.py file and added additional requirements needed to run the test notebooks (mainly visualization requirements: umap and plotnine)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.
+pytest
+nbval
+umap-learn
+plotnine
```
Comment (ben-heil) [inline requirements.txt:5]: Are pandas, numpy, etc not required?

Typically a requirements file has more detail about what is required, like in an evironment.yml file. (See [this link](https://packaging.python.org/discussions/install-requires-vs-requirements/#requirements-files) for more details.

Comment (ben-heil) [reply requirements.txt:5]: Also, if you have different requirements for installing and testing ponyo you can use the `tests_require` parameter [shown here](https://stackoverflow.com/a/15422703/10930590) and install ponyo with `pip install .[test]`

Comment (ajlee21) [reply requirements.txt:5]: Good point! That is confusing. The `requirements.txt` was only meant to indicate requirements for testing and the install requirements are in the `setup.py` file. 

I will update my setup.py accordingly

**Comment by ajlee21** [comment]
Made some updates..let me know if you have other comments!

**Pull request merged**